yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/IbvOptions.java

```diff
@@ -0,0 +1,60 @@
+package com.yoti.api.client.docs.session.create;
+
+import com.yoti.api.client.docs.DocScanConstants;
+import com.yoti.api.client.spi.remote.util.Validation;
+
+public class IbvOptions {
+
+    private final String support;
+    private final String guidanceUrl;
+
+    private IbvOptions(String support, String guidanceUrl) {
+        this.support = support;
+        this.guidanceUrl = guidanceUrl;
+    }
+
+    public static IbvOptions.Builder builder() {
+        return new IbvOptions.Builder();
+    }
+
+    public String getSupport() {
+        return support;
+    }
+
+    public String getGuidanceUrl() {
+        return guidanceUrl;
+    }
+
+    public static class Builder {
+
+        private String support;
+        private String guidanceUrl;
+
+        private Builder() {}
+
+        public Builder withNotAllowedSupport() {
```

Comment:
Does it read better to say _withIbvNotAllowed_?

Reply:
Fixed